src/dashboard/src/installer/forms.py

```diff
@@ -22,8 +22,21 @@
 from django.forms.widgets import TextInput
 from django.utils.translation import ugettext_lazy as _l
 
+from main.models import DashboardSetting
+
+
+site_url_field = forms.CharField(
```

Comment:
Could we use `URLField` to improve validation when it's not read-only?

Reply:
I wanted to but `URLField` won't validate values like "http://archivematica-storage-sevice:8000`. I guess it's looking for fully qualified domain names but if you're on a local network I'm not sure why that should be a requirement.